services/repository/files/file_test.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 func TestCleanUploadFileName(t *testing.T) {
-	assert.Equal(t, "", CleanGitTreePath(""))  //nolint
-	assert.Equal(t, "", CleanGitTreePath(".")) //nolint
+	assert.Empty(t, CleanGitTreePath(""))
+	assert.Empty(t, CleanGitTreePath("."))
 	assert.Equal(t, "a/b", CleanGitTreePath("a/b"))
-	assert.Equal(t, "", CleanGitTreePath(".git/b")) //nolint
-	assert.Equal(t, "", CleanGitTreePath("a/.git")) //nolint
+	assert.Empty(t, CleanGitTreePath(".git/b"))
+	assert.Empty(t, CleanGitTreePath("a/.git"))
```

Comment:
It was also written by me

I believe `assert.Equal(t, ",` is more readable than sometimes `assert.Empty` but sometimes `assert.Equal`.

The  `assert.Equal(t, ",` lines just align with others. 

That's another case why I dislike some lints.